creator-node/src/config.js

```diff
@@ -344,24 +344,12 @@ const config = convict({
     env: 'spOwnerWalletIndex',
     default: 0
   },
-  isUserMetadataNode: {
-    doc: 'Flag indicating whether to run this node for user metadata (non creators) only',
-    format: Boolean,
-    env: 'isUserMetadataNode',
-    default: false
-  },
   isReadOnlyMode: {
     doc: 'Flag indicating whether to run this node in read only mode (no writes)',
     format: Boolean,
     env: 'isReadOnlyMode',
     default: false
   },
-  userMetadataNodeUrl: {
```

Comment:
looks like this config is present in all k8s as well (don't need to remove it ofc but would be good to cleanup together)

Reply:
deleted from k8s config map